components/micro-gateway-cli/src/main/resources/templates/service.mustache

```diff
@@ -316,7 +316,6 @@ service {{cut qualifiedServiceName " "}} on {{#api.transport}}{{#equals . "http"
                                 message : "Runtime Error",
                                 description : errorDescription
                             }};
-
                             res.setPayload(payload);
```

Comment:
better to keep the empty line to separate this line from payload creation and make it more readable.
```suggestion

                res.setPayload(payload);
```